### PR TITLE
Send converted UTC date for completedTask date field

### LINKED ISSUE #131: Update completedTask date to use UTC date (time set to 00:00:000)
Will explain in details in Notion why we are doing this...

### PR DESCRIPTION
**Pull request by kome12**
## Description

Updated daily gardening's API endpoint to send the local user's date to the backend so it will filter out the completedTask for today.

## Closes issue(s)

Closes #131 

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code

### CODE REVIEW

File: src/pages/DailyGardening/index.tsx [hunk 1]
```diff
@@ -73,17 +73,32 @@ export const DailyGardening = () => {
...
+      getGardenData(gardenId, dateISO);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [gardenId]);
 
   const completeTaskHandler = useCallback(
     async (rule: Rule) => {
+      const localeDate: Date = new Date();
+      const utcDate: Date = new Date(
+        Date.UTC(
```
Comment (daiHash) [inline src/pages/DailyGardening/index.tsx:88]: I think here no need to explicitly add the type, typescript would infer from value
image

File: src/pages/DailyGardening/index.tsx [hunk 2]
```diff
@@ -73,17 +73,32 @@ export const DailyGardening = () => {
...
+          localeDate.getFullYear(),
+          localeDate.getMonth(),
+          localeDate.getDate(),
+          0,
+          0,
+          0,
+          0
+        )
+      );
```
Comment (daiHash) [inline src/pages/DailyGardening/index.tsx:97]: 🙏

**Review by daiHash — APPROVED** [review]
Thanks for the fix LGTM!👍

**Review by curtisahuang — APPROVED** [review]

File: src/pages/DailyGardening/index.tsx (hunk 1 above)
Comment (kome12) [reply src/pages/DailyGardening/index.tsx:88]: Removed the type!

**Pull request merged**